docs/patterns.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "spectral-rulesets",
+  "name": "spectral",
   "version": "1.16.0",
```

Comment:
Shouldn't we report the actual [Spectral tool version 6.7.0](https://github.com/stoplightio/spectral/releases/tag/v6.7.0) instead?

Reply:
The problem is getting that version programmatically. I don't know how.